cmd/root_test.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	exitCode := m.Run()
+
+	os.Exit(exitCode)
+}
```

Comment:
Since you're not doing any setup or teardown in this function it is unnecessary and can be deleted.